runtime-interface/src/lib.rs

```diff
@@ -56,7 +56,9 @@ pub struct LazyPagesProgramContext {
     pub wasm_mem_size: u32,
     /// Wasm program stack end page.
     pub stack_end: Option<u32>,
-    /// Wasm program id.
+    /// The field contains prefix to a program's memory pages, i.e.
+    /// `program_id` + `memory_infix`.
+    /// For backward compatibility it has not been renamed.
```

Comment:
backward compatibility won't be broken by renaming cause namings are not define approach of encoding